summary-visualiser/templates/scenarios/remote_call_rate.html.tmpl

```diff
@@ -0,0 +1,74 @@
+{{- define "description" }}
+<p>Test the throughput of <code>remote_call</code> operations. This scenario reads the environment variable <code>MIN_PEERS</code> and waits for at least that many agents to be available before starting the agent behaviour. It will wait up to two minutes, then proceed regardless. Note that the number of agents seen by each conductor includes itself, so having two conductors means that each will immediately see one agent after app installation.</p>
```

Comment:
To me, the details about MIN_PEERS is a bit of an implementation thing. It's been done that way so that as much as possible, the data isn't affected by the initial peer discovery and setup phase. You'd have to dig pretty deep into the data to care about that.